distributed/bokeh/scheduler_html.py

```diff
@@ -163,6 +163,22 @@ def get(self):
         self.write(result)
 
 
+class PrometheusHandler(RequestHandler):
+    def get(self):
+        import prometheus_client
+        workers = prometheus_client.Gauge('workers_total',
```

Comment:
I'm not sure about defining this in here. Same for the other one below. Creating this object automatically registers it with the prometheus client. I could see this causing conflicts when accessing multiple times.

Reply:
You're right about this, will move to class init.